sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -1275,6 +1303,7 @@ class RapidsShuffleInternalManagerBase(conf: SparkConf, val isDriver: Boolean)
               mapId,
               conf,
               new ThreadSafeShuffleWriteMetricsReporter(metricsReporter),
+              rapidsConf.shuffleMultiThreadedMaxBytesInFlight,
```

Comment:
one thing I'd add in this PR is a change in the documentation for this conf. It currently says it is only for deserialization, but with this change we have used it for serialization as well.